base/markdown/render/plain.jl

```diff
@@ -37,6 +37,46 @@ function plain(io::IO, list::List)
     end
 end
 
+function plain(io::IO, md::Table)
+    col_widths = reduce(max, map(cell -> map(ansi_length, cell), md.rows))
```

Comment:
If the plain text has ansi codes in, something's gone wrong ;) `length` should be fine here, and you'll want to render each cell beforehand as well.